### ISSUE TITLE
Matrix icon not showing up properly on Firefox

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#205]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

The left bracket of the Matrix icon isn’t showing up properly on Firefox using Linux. Would be good to switch it over to another icon that has a bit brackets that are a bit thicker to make sure that it doesn’t disappear :)

This is a great `good first issue` for someone who wants to contribute to activist for the first time! Happy to detail where the changes need to be made as well 😊

**Comment by Daseats96** [comment]
Hello, I'd be happy to give this a try as this would be my first contribution.

**Comment by andrewtavis** [comment]
Hey @Daseats96! @eloi-cht has a PR open for this already and is going to be working on it :) Thanks for checking in! I'll try to make some more issues in the coming days 🙃 Will also write you back on Matrix about picking up tasks that are already claimed or have been sitting for a moment.

**Comment by Daseats96** [comment]
sounds good, if any other tickets are created please feel free to ping me.

**Comment by andrewtavis** [comment]
If you'd like to take a look at #211 and take some of the files there that'd be great, @Daseats96!

**Comment by eloi-cht** [comment]
Hey @andrewtavis,

After a bit more research and an interesting read on [SVG scaling](https://css-tricks.com/scale-svg/), the cleanest way to do it I found is adding 2 pixels to the viewbox which leave some extra space for the SVG to scale without getting cropped.

I played around with the attributes to maintain original logo size and keep it centered and these works well on my end
size="1.061em"
viewBox="-1 -1 34 34"

I'm not sure what the best way to implement it, I tried setting viewBox directly using nuxt-icon but it is ignored and it uses Iconify viewBox instead.
I also tried creating a Vue component and call it which works fine but requires to have SVG locally.

I'd be happy to hear what you think about this, I've updated the branch with the solution that works. :)
